modules/basegl/src/processors/imageprocessing/columnrowlayout.cpp

```diff
@@ -341,6 +341,7 @@ void ColumnLayout::updateViewports(ivec2 dim) {
     for (auto i = 0; i < std::max(1, numViews); ++i) {
         viewManager_.push_back(ivec4(positions[i], 0, positions[i + 1] - positions[i], dim.y));
     }
+    notifyObserversActiveConnectionsChange(this);
```

Comment:
Should this not be placed here?
```
    inport_.onConnect([this]() { updateSplitters(true); });
    inport_.onDisconnect([this]() { updateSplitters(false); });
```

Reply:
`updateViews()` is also called in `updateSliders()`